### ISSUE TITLE
Dashboard: Landmark markup

### ISSUE THREAD

**Issue by tskarhed** [grafana/grafana#36647]
**What would you like to be added**:
Change a bunch of divs to semantic HTML components such as main, header, section.

- Dashboard header
  - [ ] Should be `heading`
  - [ ] The name should be `h1` and `nav` (Can these conflict somehow?)
- Panels
  - [ ] See them as `section`.
  - [ ] Panel titles should be `h2`
  - [ ] Titles within the panel should be `h3`
- [ ] Remove label-only divs and replace with semantic elements or other ARIA attribute 

**Why is this needed**:

To make the dashboard more screen reader friendly.

**Comment by Elfo404** [comment]
Regarding
> Panel titles should be h2
Titles within the panel should be h3

I think we are not considering rows, in that case:

- Rows titles should be h2 (and rows should create a `section`)
- Panel titles should be h3
- Titles within the panel should be h4

WDYT?

**Comment by Elfo404** [comment]
also this should probably be `main` instead of `div`?

https://github.com/grafana/grafana/blob/4cbffae1b459ef14bebbf812edae49b9dd94bab3/public/app/AppWrapper.tsx#L100

**Comment by tskarhed** [comment]
> Regarding
> 
> > Panel titles should be h2
> > Titles within the panel should be h3
> 
> I think we are not considering rows, in that case:
> 
>     * Rows titles should be h2 (and rows should create a `section`)
> 
>     * Panel titles should be h3
> 
>     * Titles within the panel should be h4
> 
> 
> WDYT?

Some accessibility tools aren't too happy if you jump directly from an h1 to an h3, but I think it's a good solution.

**Comment by Elfo404** [comment]
> Some accessibility tools aren't too happy if you jump directly from an h1 to an h3

that's actually a very good point I didn't think of. i wonder if we can find a solution suitable for both cases
